WikiWatershed/model-my-watershed
# Display stream concentration values in Water Quality analysis tab

This PR updates the Water Quality analysis tab and popup to display catchment values for the new concentration columns added in #1531. The output currently looks like this:

<img width="1263" alt="screen shot 2016-10-12 at 5 28 41 pm" src="https://cloud.githubusercontent.com/assets/4165523/19328461/b2daa954-90a1-11e6-8aa4-f537013b1ee5.png">

& ->

<img width="1171" alt="screen shot 2016-10-12 at 5 23 27 pm" src="https://cloud.githubusercontent.com/assets/4165523/19328468/ba0d0852-90a1-11e6-89de-d3ed756ff6e7.png">

The first commit here is a shim css change to adjust the tab width so that the water quality tab will display. The second adjusts `calcs.py` to retrieve the new columns from the `drb_catchment_water_quality` table and return them with the analysis results, and updates the frontend ui to add logic and templates for displaying the new columns in the table added in #1526 and a utility function to return the average of the concentration values, skipping ones which were `null`.
 
**Testing**
- get this branch, `vagrant up`, `vagrant ssh app` then `cd /vagrant/scripts/aws` and run `./scripts/setupdb.sh -q` to get the updated water quality table
- visit `localhost:8000` in the browser and open the browser console
- select an AOI within the DRB. When the analysis results return verify that you see the new columns in the water quality tab and that when you click on one of the rows' ids, the new values are also visible in the popup.

Connects #1532
